Author: Michael R. Crusoe <crusoe@debian.org>
Description: Adjust dependencies to match what is packaged for Debian
Forwarded: Not-needed
--- piz.orig/Cargo.toml
+++ piz/Cargo.toml
@@ -67,7 +67,7 @@
 version = "1.0"
 
 [dev-dependencies.env_logger]
-version = "0.8"
+version = "0.11"
 
 [dev-dependencies.memmap]
 version = "0.7"
